lib/components/user/existing-account-display.js

```diff
@@ -1,43 +1,42 @@
-import React, { Component } from 'react'
+import React from 'react'
 
 import LinkButton from './link-button'
 import StackedPaneDisplay from './stacked-pane-display'
 
 /**
  * This component handles the existing account display.
  */
-class ExistingAccountDisplay extends Component {
-  render () {
-    const { onCancel, onComplete, panes } = this.props
-    const paneSequence = [
-      {
-        pane: () => <p><LinkButton to='/savedtrips'>Edit my trips</LinkButton></p>,
-        title: 'My trips'
-      },
-      {
-        pane: panes.terms,
-        props: { disableCheckTerms: true },
-        title: 'Terms'
-      },
-      {
-        pane: panes.notifications,
-        title: 'Notifications'
-      },
-      {
-        pane: panes.locations,
-        title: 'My locations'
-      }
-    ]
+const ExistingAccountDisplay = props => {
+  const { onCancel, panes } = props
+  const paneSequence = [
+    {
+      pane: () => <p><LinkButton to='/savedtrips'>Edit my trips</LinkButton></p>,
+      title: 'My trips'
+    },
+    {
+      pane: panes.terms,
+      props: { disableCheckTerms: true, ...props },
+      title: 'Terms'
+    },
+    {
+      pane: panes.notifications,
+      props,
+      title: 'Notifications'
+    },
+    {
+      pane: panes.locations,
+      props,
+      title: 'My locations'
+    }
+  ]
 
-    return (
-      <StackedPaneDisplay
-        onCancel={onCancel}
-        onComplete={onComplete}
-        paneSequence={paneSequence}
-        title='My Account'
-      />
-    )
-  }
+  return (
+    <StackedPaneDisplay
+      onCancel={onCancel}
+      paneSequence={paneSequence}
+      title='My Account'
```

Comment:
Generally, the titles in these pages are in sentence case (first word capitalized) rather than title case. Given that, I think we should change this title to `My account`.

Reply:
Will not address. Internal reviews are recommending `My Account` instead.